chat/src/test/java/RoomBaseActionTestIT.java

```diff
@@ -0,0 +1,194 @@
+import static com.codeborne.selenide.Selectors.byText;

```

Comment:
This file must be located in a package `package org.exoplatform.platform.qa.ui.chat;`